src/sentry/static/sentry/app/views/organizationDashboard/utils/getEventsUrlPathFromDiscoverQuery.jsx

```diff
@@ -13,8 +14,8 @@ export function getEventsUrlPathFromDiscoverQuery({organization, selection, quer
   return `/organizations/${organization.slug}/events/?${qs.stringify(
     pickBy({
```

Comment:
What's `pickBy` without a predicate function - isn't that just `identity`?

Reply:
Yeah, so will return pick only values that are truthy